politeiawww/api/cms/v1/v1.go

```diff
@@ -433,3 +446,37 @@ type PaymentInformation struct {
 	AmountReceived  dcrutil.Amount `json:"amountreceived"`
 	Status          PaymentStatusT `json:"status"`
 }
+
+// CMSUser represents a CMS user. It contains the standard politeiawww user
+// fields as well as CMS specific user fields.
+type CMSUser struct {
+	User               www.User        `json:"user"`
```

Comment:
`www.User` includes a bunch of fields that cms doesn't use.  Might it be better to have the `CMSUser` defined as just the fields that cms uses so that the api is not returning unnecessary fields?

Also something to consider.  This is in the cms api package so when its being referenced throughout the politeiawww code its being referenced as `cms.CMSUser`.  The `cms.CMS` is redundant.  Just using `cms.User` would be fine.

Reply:
Yah I wasn't sure about that.  While we may not be using that information currently, I figured that there could be some existing `UserDetails` calls (pigui for example) that need to get some of the current fields in `www.User`  (isAdmin, email, username, identities etc).   Would you rather I just add the various fields that we may use rather than reuse `www.User`?

Will be making the change concerning the 2nd point for consideration.

Reply:
Yeah, I'd rather you just add the fields that cms uses.  If it breaks anything in the gui, those are probably things that need to be update/fixed anyway.

Reply:
OK will do.